packages/nextjs/services/web3/stark-burner/BurnerConnector.ts

```diff
@@ -59,11 +59,12 @@ export class BurnerConnector extends InjectedConnector {
       new Account(
         new RpcProvider({
           nodeUrl: this.chain.rpcUrls.public.http[0],
-          chainId: starknetChainId(this.chain.id),
+          // chainId: starknetChainId(this.chain.id), // TODO : check how this affects wrong network. can we get this from the chain directly ?
```

Comment:
wdym by getting from the chain directly?

Reply:
yes with the workaround of `getChainId()`